liveblog.php

```diff
@@ -577,15 +577,15 @@ public static function ajax_lazyload_entries() {
 
 		// Get all Liveblog entries that are to be lazyloaded.
 		$entries = self::$entry_query->get_for_lazyloading(
-			isset( $fragments[1] ) ? (int) $fragments[1] : 0,
-			isset( $fragments[2] ) ? (int) $fragments[2] : 0
+			isset( $fragments[2] ) ? (int) $fragments[2] : 0,
+			isset( $fragments[3] ) ? (int) $fragments[3] : 0
 		);
 		if ( ! $entries ) {
 			do_action( 'liveblog_entry_request_empty' );
 
 			self::json_return( array(
 				'entries' => array(),
-				'index'   => (int) filter_input( INPUT_GET, 'index' ),
+				'index'   => (int) ( isset( $fragments[1] ) ? $fragments[1] : 0 ),
```

Comment:
What happens if this gets set to `0`? How does the code deal with that eventually, and what is the effect for the user?